crates/viewer/re_view_graph/src/layout/provider.rs

```diff
@@ -161,24 +180,32 @@ impl ForceLayoutProvider {
         let simulation = update_simulation(simulation, params, edges, radii);
 
         Self {
-            simulation,
+            simulation: Some(simulation),
             request,
         }
     }
 
     fn layout(&self) -> Layout {
         // We make use of the fact here that the simulation is stable, i.e. the
         // order of the nodes is the same as in the `request`.
-        let mut positions = self.simulation.positions();
+        let positions: &mut dyn Iterator<Item = Pos2> = if let Some(simulation) = &self.simulation {
+            &mut simulation
+                .positions()
+                .map(|[x, y]| Pos2::new(x as f32, y as f32))
+        } else {
+            &mut self.request.all_nodes().map(|(_, v)| {
+                v.fixed_position
+                    .expect("if there is no simulation, all nodes have to be fixed")
```

Comment:
Don't use `expect` in release builds. If something can crash, it will 100% crash at some point (and take the entire viewer as well as all the user's unsaved data with it).

This could be a `.map` + `expect()` in debug, and a `.filter_map` + `error!` log in release.